resource/graph_node.go

```diff
@@ -539,15 +529,6 @@ func (w *GraphNode) ResourceStatus() Status {
 	return w.resourceStatus()
 }
 
-func (w *GraphNode) getLoggerOrGlobal() logging.Logger {
```

Comment:
@maximpertsov I removed this and nothing broke so will take it and its usage out unless you disagree.

Reply:
yep! it's possible that we never needed to guard against a `nil` logger for this code path in the first place. I'm curious if we can remove the remaining if `w.logger != nil` guards in this file. But no need to do that yet, this change is large enough as is.